modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/TestrayRouter.tsx

```diff
@@ -332,7 +333,17 @@ const TestrayRoute = () => (
 						<Route element={<CompareRunsTeams />} path="teams" />
 					</Route>
 
-					<Route element={<div>Page not found</div>} path="*" />
+					<Route
+						element={
+							<PageRedirect
+								errorMessage="This record does not exist"
+								errorTitle="error"
+								errorType="EMPTY_SEARCH"
+								pageToRender="error"
+							/>
+						}
+						path="*"
+					/>
```

Comment:
I added this here: https://github.com/pt-liferay-solutions/liferay-portal/pull/3839/commits/54a4497cda4d8cac7fa7826c8999e8cf8a31dd4a

Reply:
resolved, thank you